src/main/webapp/app/app.scss

```diff
@@ -429,3 +429,14 @@ a {
 .toastify-toast {
   display: inline-flex;
 }
+
+.scrollbar-wrapper {
```

Comment:
Does this have to be a global style?

Reply:
Originally only had it in navigation sidebar scss, but figured that there could be other components that we might want to add this class.